elastic/kibana
# [Telemetry] Pull local Kibana usage stats

Closes https://github.com/elastic/kibana/issues/21241

## Summary

This change makes telemetry stats for Kibana available via getting pulled live. Before now, the stats had to be pulled from Monitoring data. 

## Testing

1. Enable Monitoring
2. View an example of telemetry data 
   -  Copy the data as `monitoringEnabled.json`
4. Disable Monitoring and remove all Monitoring data
5. View an example of telemetry data again
   - Copy the data as `monitoringDisabled.json`
6. Compare `monitoringEnabled[0].stack_stats.kibana` vs `monitoringDisabled[0].stack_stats.kibana`

The same information about Kibana should be in both captures. 

This PR has no other effects that users will interact with.
